### PR TITLE
Feature/create and link shelf mark sequence

### PR DESCRIPTION
**Pull request by kwahlin**
With get-active-shelf-mark-seqs.sh we retrieve current state (including shelf control number) of the shelf mark sequences from the old database. Note that mssql-tools needs to be installed and working on the machine where we run it. 

The whelktool script then creates the shelf mark sequences and also links holdings that use them.

### CODE REVIEW

File: whelktool/scripts/2021/03/create-and-link-shelf-mark-seqs/should-be-active.csv [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Ry 2021 Q,Ryska monografier,Kvarto
+Ry 2021 F,Ryska monografier,Folio
+Sv2021,Svenska monografier,Oktav
+Sv2021Q,Svenska monografier,Kvarto
+Sv2021F,Svenska monografier,Folio
+Sv2021 br,Svenska broschyrer,Oktav
+Sv2021Q br,Svenska broschyrer,Kvarto
+Sv2021F br,Svenska broschyrer,Folio
+Sv2021 Diss,Svenska dissertation,Oktav
+Sv2021Q Diss,Svenska dissertation,Kvarto
```
Comment (olovy) [inline whelktool/scripts/2021/03/create-and-link-shelf-mark-seqs/should-be-active.csv:17]: dissertationer

File: whelktool/scripts/2021/03/create-and-link-shelf-mark-seqs/should-be-active.csv [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Ry 2021,Ryska monografier,Oktav
+Ry 2021 Q,Ryska monografier,Kvarto
+Ry 2021 F,Ryska monografier,Folio
+Sv2021,Svenska monografier,Oktav
+Sv2021Q,Svenska monografier,Kvarto
+Sv2021F,Svenska monografier,Folio
+Sv2021 br,Svenska broschyrer,Oktav
+Sv2021Q br,Svenska broschyrer,Kvarto
+Sv2021F br,Svenska broschyrer,Folio
+Sv2021 Diss,Svenska dissertation,Oktav
```
Comment (olovy) [inline whelktool/scripts/2021/03/create-and-link-shelf-mark-seqs/should-be-active.csv:16]:
```suggestion
Sv2021 Diss,Svenska dissertationer,Oktav
```

File: whelktool/scripts/2021/03/create-and-link-shelf-mark-seqs/should-be-active.csv (hunk 1 above)
Comment (olovy) [inline whelktool/scripts/2021/03/create-and-link-shelf-mark-seqs/should-be-active.csv:17]:
```suggestion
Sv2021 Diss,Svenska dissertationer,Oktav
Sv2021Q Diss,Svenska dissertationer,Kvarto
```

Comment (kwahlin) [reply whelktool/scripts/2021/03/create-and-link-shelf-mark-seqs/should-be-active.csv]: Fixed!

**Pull request merged**